function-images/tests/producer-consumer/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM amd64/golang:1.15.7-alpine3.13 as consumer
```

Comment:
interesting... is it recommended to define several images in a single file?

Reply:
I don't know if it's recommended, but I'm not sure if it will be possible to use docker-compose if there are multiple docker files scattered in subfolders